src/client/App.css

```diff
@@ -41,12 +41,16 @@ ul {
   list-style: none;
 }
 
-.registrationPage h1 {
-  margin-bottom: 10vh;
+.loginPage h1 {
```

Comment:
Why did you change this to loginPage? Will is not interfere with the registration styling?

Reply:
Registration page does not have an h1 anymore, this was left over from when it is and I simply reused what was already there for the login page